bundles/org.openhab.binding.ism8/src/main/java/org/openhab/binding/ism8/internal/Ism8Handler.java

```diff
@@ -0,0 +1,158 @@
+                    && channel.getConfiguration().get("write").toString().equalsIgnoreCase("true")) {
+                int id = Integer.parseInt(channel.getConfiguration().get("id").toString());
+                this.logger.debug("Channel '{}' writting into ID '{}'", channel.getUID().getId(), id);
+                this.updateState(channelUID, new QuantityType<>(command.toString()));
+
+                IDataPoint dataPoint = null;
+                for (IDataPoint dp : svr.getDataPoints()) {
+                    if (dp.getId() == id) {
+                        dataPoint = dp;
+                        break;
+                    }
+                }
+
+                if (dataPoint != null) {
+                    try {
+                        svr.sendData(dataPoint.createWriteData(command));
+                    } catch (IOException e) {
+                        this.logger.warn("Writting to ISM DataPoint '{}' failed. '{}'", dataPoint.getId(),
+                                e.getMessage(), e);
+                    }
+                }
+            }
+        }
+    }
+
+    @Override
+    public void dispose() {
+        Server svr = this.server;
+        if (svr != null) {
+            svr.stopServerThread();
+        }
+    }
+
+    @Override
+    public void initialize() {
+        this.config = getConfigAs(Ism8Configuration.class);
+        try {
+            Ism8Configuration cfg = this.config;
+            if (cfg != null) {
+                Server svr = new Server(cfg.getPortNumber());
+                for (Channel channel : getThing().getChannels()) {
+                    if (channel.getConfiguration().containsKey("id")
+                            && channel.getConfiguration().containsKey("type")) {
+                        int id = Integer.parseInt(channel.getConfiguration().get("id").toString());
+                        String type = channel.getConfiguration().get("type").toString();
+                        String description = channel.getLabel();
+                        if (type != null && description != null) {
+                            svr.addDataPoint(id, type, description);
+                        }
+                    } else {
+                        this.logger.warn("Ism8: ID or type missing - Channel={}  Cfg={}", channel.getLabel(),
+                                channel.getConfiguration());
+                    }
+                    this.logger.debug("Ism8: Channel={}", channel.getConfiguration().toString());
+                }
+
+                this.updateStatus(ThingStatus.UNKNOWN);
+                svr.addDataPointChangeListener(this);
+                scheduler.execute(svr::start);
+                this.server = svr;
+            }
+        } catch (Exception e) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, e.getMessage());
+            this.logger.warn("Ism8 Initialize: {}", e.getMessage(), e);
```

Comment:
The message is already logged by the framework, so this log message can be removed. And logging the stack trace seems a bit unreasonable.